bin/user/2FA/index.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const Cli = require('structured-cli');
+
+const crypto = require('crypto');
+
+const options = {
+    type: {
+        description: 'type'
+      , type: 'string'
+      , default: 'totp'
+    }
+};
+
+const enableTOTP = async (resource, args) => {
+    const interactive = require('lib/interactive');
```

Comment:
Why do you load it there?

Reply:
Check those two lines after. I'm only loading  _speakeasy_ and _qrcode_ and they are needed. _interactive_ is just to keep it clean since it's already in memory.

Reply:
Thanks for the clarification. I was interested only in ``interactive``, because I was aware that it was already in memory.